po/wesnoth/de.po

```diff
@@ -8767,105 +8444,105 @@ msgstr ""
 "Durch Gift allein können Einheiten allerdings nicht sterben. Eine Vergiftung "
 "wird die LP einer Einheit nicht unter 1 reduzieren."
 
-#: src/reports.cpp:352
+#: src/reports.cpp:348
 msgid "petrified: "
 msgstr "versteinert: "
 
-#: src/reports.cpp:353
+#: src/reports.cpp:349
 msgid "This unit has been petrified. It may not move or attack."
 msgstr ""
 "Diese Einheit wurde zu Stein verwandelt. Sie kann sich weder bewegen noch "
 "angreifen."
 
-#: src/reports.cpp:356
+#: src/reports.cpp:352
 msgid "unhealable: "
 msgstr "unheilbar: "
 
-#: src/reports.cpp:357
+#: src/reports.cpp:353
 msgid ""
 "This unit is unhealable. It cannot be healed by healers or villages and "
 "doesn’t benefit from resting."
 msgstr ""
 "Diese Einheit ist unheilbar. Sie kann weder von Heilern noch in Dörfern "
 "geheilt werden, und erholt sich nicht, wenn sie sich ausruht."
 
-#: src/reports.cpp:360
+#: src/reports.cpp:356
 msgid "invulnerable: "
 msgstr "unverwundbar: "
 
-#: src/reports.cpp:361
+#: src/reports.cpp:357
 msgid "This unit is invulnerable. It cannot be harmed by any attack."
 msgstr ""
 "Diese Einheit ist unverwundbar. Sie erleidet von keinem Angriff Schaden."
 
-#: src/reports.cpp:438
+#: src/reports.cpp:434
 msgid "Ability: "
 msgstr "Fertigkeit: "
 
-#: src/reports.cpp:440 src/reports.cpp:1013
+#: src/reports.cpp:436 src/reports.cpp:1009
 msgid " (inactive)"
 msgstr " (inaktiv)"
 
-#: src/reports.cpp:504
+#: src/reports.cpp:500
 msgid "Resistances: "
 msgstr "Resistenzen: "
 
-#: src/reports.cpp:506
+#: src/reports.cpp:502
 msgid "(Att / Def)"
 msgstr "(Ang / Ver)"
 
-#: src/reports.cpp:535
+#: src/reports.cpp:531
 msgid "Experience Modifier: "
 msgstr "Erfahrungsmodifikator: "
 
-#: src/reports.cpp:585
+#: src/reports.cpp:581
 msgid "Terrain: "
 msgstr "Gelände: "
 
-#: src/reports.cpp:602
+#: src/reports.cpp:598
 msgid "maximum^max."
 msgstr "max."
 
-#: src/reports.cpp:602
+#: src/reports.cpp:598
 msgid "minimum^min."
 msgstr "min."
 
-#: src/reports.cpp:607
+#: src/reports.cpp:603
 msgid "Defense: "
 msgstr "Verteidigung: "
 
 # Oder auch nur Sicht
-#: src/reports.cpp:642 src/reports.cpp:643
+#: src/reports.cpp:638 src/reports.cpp:639
 msgid "vision:"
-msgstr "👁"
+msgstr "Sicht:"
```

Comment:
Will be displayed in the sidebar as "Sic." – thus the emoticon instead.

Reply:
Factor of #1167 and related issues.

Reply:
> Factor of #1167 and related issues.

#6021 more specifically, I think